### PR TITLE
Example fix

### PR DESCRIPTION
**Pull request by alauper**
this doesn't look like valid code and doesn't use the values parameter sent in, which I think was the point of the example.

### CODE REVIEW

**Comment by alauper** [comment]
There's actually a few ambiguities with that example.

It's not clear where ageValidation is being called from.  Are the passed in parameters "value, values" or "value, allValues"?  And do you return undefined or []?

File: docs/CreateEdit.md [hunk 1]
```diff
@@ -321,7 +321,7 @@ const ageValidation = (value, values) => {
     if (!value) {
         return 'The age is required';
     }
-    if (age < 18) {
+    if (values.age < 18) {
```
Comment (djhi) [inline docs/CreateEdit.md]: Agreed, this example is wrong. About your question, the parameters are `(value, allValues)`. So here, `age` should be `value`.

File: docs/CreateEdit.md [hunk 2]
```diff
@@ -317,11 +317,11 @@ const number = value => value && isNaN(Number(value)) ? 'Must be a number' : und
 const minValue = min => value =>
   value && value < min ? `Must be at least ${min}` : undefined;
 
-const ageValidation = (value, values) => {
+const ageValidation = (value, allValues) => {
     if (!value) {
         return 'The age is required';
     }
-    if (age < 18) {
+    if (allValues.age < 18) {
```
Comment (djhi) [inline docs/CreateEdit.md]: I meant `allValues.age` should simply be `value`.

Comment (alauper) [reply docs/CreateEdit.md]: I actually thought allValues.age would demonstrate how to use the allValues parameter.  But I agree with your opinion too.  I updated the commit.  Thanks!

**Pull request merged**